custom-vu/src/main/kotlin/jces1209/vu/page/IssueNavigator.kt

```diff
@@ -1,17 +1,84 @@
 package jces1209.vu.page
 
 import com.atlassian.performance.tools.jiraactions.api.WebJira
+import jces1209.vu.page.ViewSubscriptions.ViewSubscriptions
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.openqa.selenium.support.ui.ExpectedConditions.*
 
 abstract class IssueNavigator(
-    private val jira: WebJira
+    protected val jira: WebJira
 ) {
     protected val driver = jira.driver
-    
+    protected val filterSubscriptionList = and(
+        presenceOfElementLocated(By.id("filter-subscription")),
+        presenceOfElementLocated(By.id("filter.subscription.prefix.daysOfMonth")),
+        presenceOfElementLocated(By.className("form-body")),
+        presenceOfElementLocated(By.xpath("//*[. = 'Recipients']")),
+        presenceOfElementLocated(By.xpath("//*[. = 'Schedule']")),
+        presenceOfElementLocated(By.xpath("//*[. = 'Interval']")),
+        presenceOfElementLocated(By.id("filter-subscription-submit")),
+        presenceOfElementLocated(By.id("filter-subscription-cancel"))
+    )
+    protected abstract val filterSubscriptionFalliblePage: FalliblePage
+    protected abstract val viewSubscriptions: ViewSubscriptions
+
     abstract fun waitForNavigator()
     abstract fun selectIssue()
 
+    fun clickNewSubscription() {
+        driver.wait(
+            elementToBeClickable(By.xpath("//*[@id = 'inline-dialog-filter-details-overlay']//*[contains(text(), 'New subscription')]"))
+        )
+            .click()
+        filterSubscriptionFalliblePage.waitForPageToLoad()
+    }
+
+    open fun clickDetails() {
+        driver.wait(
+            elementToBeClickable(By.xpath("//*[@class = 'show-filter-details']"))
```

Comment:
By.className
also, what about to extract the locator - it is used in other methods already?